docs/app/pages/components/components-sections/select/select/select.component.less

```diff
@@ -0,0 +1,18 @@
+.custom-icon {
```

Comment:
We could add these rules to the library, and document it as a kind of helper class (ux-select-custom-icon) perhaps. It would save some copy/pasting.